rate_limiter/src/lib.rs

```diff
@@ -410,12 +410,10 @@ impl RateLimiter {
             ops_complete_refill_time_ms,
         );
 
-        // TODO: Self::timer_fd should not be an `Option` anymore; clean that up.
-        //
         // We'll need a timer_fd, even if our current config effectively disables rate limiting,
         // because `Self::update_buckets()` might re-enable it later, and we might be
         // seccomp-blocked from creating the timer_fd at that time.
-        let timer_fd = Some(TimerFd::new_custom(ClockId::Monotonic, true, true)?);
+        let timer_fd = TimerFd::new_custom(ClockId::Monotonic, true, true)?;
```

Comment:
Since this is the case now, do a quick experiment and run the CI with `create_timerfd()` **not** whitelisted in seccomp and see if we can remove it from the seccomp.

Reply:
Ran the experiment. Tests fail. We should look into it later.